gossipsub.go

```diff
@@ -3,6 +3,7 @@ package pubsub
 import (
 	"context"
 	"fmt"
+	"github.com/libp2p/go-libp2p-core/record"
```

Comment:
can you put this together with our other imports? Here it is mixed with system imports.